lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -215,12 +215,16 @@ int main(int argc, char* argv[]) {
       MAX_URL_LENGTH);
   initialize_sentry(SENTRY_TAG_SESSIOND, &sentry_config);
 
-  bool converged_access          = false;
+  bool enable5g_features         = false;
```

Comment:
lets use the cpp format here
```
enable_5g_features
```

`config["enable5g_features"]` is fine because it comes from swagger that comes from golang and it is a flagm, not a cpp var

Reply:
Yeah modified the cpp Identifier from 'enable5g_features' to 'enable_5g_features' as you suggested.